packages/shared/lib/wallet.ts

```diff
@@ -480,24 +480,13 @@ export async function asyncSyncAccountOffline(account: WalletAccount): Promise<v
     })
 }
 
-export const asyncGetNodeInfo = (accountId: string, url?: string, auth?: IAuth): Promise<INodeInfo> => {
+export const asyncGetNodeInfo = (url?: string, auth?: IAuth): Promise<IStardustNodeInfo> => {
     if (!url || (!url && !auth)) {
```

Comment:
This method takes the url and auth as optional so why do we have this guard? If these are passed in then we will just get the current nodes info.